keras/src/backend/openvino/numpy.py

```diff
@@ -910,9 +910,156 @@ def less_equal(x1, x2):
+    one_i = ov_opset.constant(1, Type.i32).output(0)
+    axis_i = ov_opset.constant(axis, Type.i32).output(0)
+
+    div = ov_opset.subtract(num, one_i).output(0) if endpoint else num
+    div = ov_opset.convert(div, dtype).output(0)
+
+    zero = ov_opset.constant(0.0, dtype).output(0)
+    one = ov_opset.constant(1.0, dtype).output(0)
+    num_f = ov_opset.convert(num, dtype).output(0)
+    seq = ov_opset.range(zero, num_f, one, dtype).output(0)
+
+    ndim = len(start.shape)
+    dims = ov_opset.concat(
+        [
+            ov_opset.constant([-1], Type.i32),
+            ov_opset.constant([1] * ndim, Type.i32),
+        ],
+        0,
+    ).output(0)
+    seq = ov_opset.reshape(seq, dims, False).output(0)
+
+    delta = ov_opset.subtract(stop, start).output(0)
+
+    cond = ov_opset.greater(div, zero).output(0)
+    nan_const = ov_opset.constant(float("nan"), dtype).output(0)
+    step_val = ov_opset.divide(delta, div).output(0)
+    step = ov_opset.select(cond, step_val, nan_const).output(0)
+
+    target_shape = ov_opset.concat(
+        [
+            ov_opset.constant([1], Type.i64),
+            ov_opset.shape_of(start).output(0),
+        ],
+        0,
+    ).output(0)
+    step = ov_opset.reshape(step, target_shape, False).output(0)
+
+    eq_zero = ov_opset.equal(step, zero).output(0)
+    any_zero = ov_opset.reduce_logical_or(
+        eq_zero, ov_opset.constant([0], Type.i32), False
+    ).output(0)
+
+    y_norm = ov_opset.multiply(seq, step).output(0)
+    y_denorm = ov_opset.multiply(
+        ov_opset.divide(seq, div).output(0),
+        delta,
+    ).output(0)
+    y_pos = ov_opset.convert(
+        ov_opset.select(any_zero, y_denorm, y_norm).output(0), dtype
+    ).output(0)
+
+    y_zero = ov_opset.convert(
+        ov_opset.multiply(seq, delta).output(0), dtype
+    ).output(0)
+    y = ov_opset.add(
+        ov_opset.convert(ov_opset.select(cond, y_pos, y_zero).output(0), dtype),
+        start,
+    ).output(0)
+
+    if endpoint:
+        idx = ov_opset.subtract(num, one_i).output(0)
+        idx = ov_opset.convert(idx, Type.i32).output(0)
+        idx_tensor = ov_opset.broadcast(idx, target_shape).output(0)
+        stop_tensor = ov_opset.broadcast(stop, target_shape).output(0)
+        y = ov_opset.scatter_elements_update(
+            y, idx_tensor, stop_tensor, 0
+        ).output(0)
+
+    if axis != 0:
+        rank = ov_opset.rank(y).output(0)
+        axis_p1 = ov_opset.add(axis_i, one_i).output(0)
+        pre = ov_opset.range(one_i, axis_p1, one_i).output(0)
+        post = ov_opset.range(axis_p1, rank, one_i).output(0)
+        zero_i = ov_opset.reshape(
+            zero_i, ov_opset.constant([1], Type.i32), False
+        ).output(0)
+        perm = ov_opset.concat([pre, zero_i, post], 0).output(0)
+        y = ov_opset.transpose(y, perm).output(0)
+
+    y = ov_opset.convert(y, out_dtype).output(0)
+    if retstep:
+        return OpenVINOKerasTensor(y), ov_opset.convert(step, out_dtype).output(
```

Comment:
create `return_step` variable and assign. `return_step` should be wrapped with `OpenVINOKerasTensor`